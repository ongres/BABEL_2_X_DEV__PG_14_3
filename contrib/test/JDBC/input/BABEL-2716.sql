use master;
go

select (case when 'a ' like 'a' then 1 else 0 end);
go
select (case when 'a ' like 'a ' then 1 else 0 end);
go
select (case when 'a  ' like 'a ' then 1 else 0 end);
go
select (case when 'a' like 'a ' then 1 else 0 end);
go
select (case when 'a ' like 'a  ' then 1 else 0 end);
go

select (case when 'a ' not like 'a' then 1 else 0 end);
go
select (case when 'a ' not like 'a ' then 1 else 0 end);
go
select (case when 'a  ' not like 'a ' then 1 else 0 end);
go
select (case when 'a' not like 'a ' then 1 else 0 end);
go
select (case when 'a ' not like 'a  ' then 1 else 0 end);
go

select (case when 'a' like 'a_' then 1 else 0 end);
go
select (case when 'a ' like 'a_' then 1 else 0 end);
go
select (case when 'a' like 'a%' then 1 else 0 end);
go
select (case when 'a ' like 'a%' then 1 else 0 end);
go
