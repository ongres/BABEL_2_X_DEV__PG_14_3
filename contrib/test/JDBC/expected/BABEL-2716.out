use master;
go

select (case when 'a ' like 'a' then 1 else 0 end);
go
~~START~~
int
1
~~END~~

select (case when 'a ' like 'a ' then 1 else 0 end);
go
~~START~~
int
1
~~END~~

select (case when 'a  ' like 'a ' then 1 else 0 end);
go
~~START~~
int
1
~~END~~

select (case when 'a' like 'a ' then 1 else 0 end);
go
~~START~~
int
0
~~END~~

select (case when 'a ' like 'a  ' then 1 else 0 end);
go
~~START~~
int
0
~~END~~


select (case when 'a ' not like 'a' then 1 else 0 end);
go
~~START~~
int
0
~~END~~

select (case when 'a ' not like 'a ' then 1 else 0 end);
go
~~START~~
int
0
~~END~~

select (case when 'a  ' not like 'a ' then 1 else 0 end);
go
~~START~~
int
0
~~END~~

select (case when 'a' not like 'a ' then 1 else 0 end);
go
~~START~~
int
1
~~END~~

select (case when 'a ' not like 'a  ' then 1 else 0 end);
go
~~START~~
int
1
~~END~~


select (case when 'a' like 'a_' then 1 else 0 end);
go
~~START~~
int
0
~~END~~

select (case when 'a ' like 'a_' then 1 else 0 end);
go
~~START~~
int
1
~~END~~

select (case when 'a' like 'a%' then 1 else 0 end);
go
~~START~~
int
1
~~END~~

select (case when 'a ' like 'a%' then 1 else 0 end);
go
~~START~~
int
1
~~END~~

